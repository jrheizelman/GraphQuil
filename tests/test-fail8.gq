/* Author: Gemma Ragozzine
 * Tests return type of main
 */ 
 
String main(){
	int j;
	j = 1;
}