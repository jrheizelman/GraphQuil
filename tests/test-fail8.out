Fatal error: exception Failure("Main method must return an void")